code/game/objects/structures/displaycase.dm

```diff
@@ -39,55 +30,53 @@
 		user << "<span class='notice'>Hooked up with an anti-theft system.</span>"
 
 
-/obj/structure/displaycase/bullet_act(obj/item/projectile/P)
-	. = ..()
-	take_damage(P.damage, P.damage_type, 0)
-
 /obj/structure/displaycase/proc/dump()
 	if (showpiece)
 		showpiece.loc = src.loc
 		showpiece = null
 
-/obj/structure/displaycase/blob_act(obj/structure/blob/B)
-	take_damage(30)
-
-/obj/structure/displaycase/hitby(atom/movable/AM)
-	..()
-	if(isobj(AM))
-		var/obj/item/I = AM
-		take_damage(I.throwforce * 0.2)

```

Comment:
Nerf, the displaycase loses its armor effect against specifically thrown items. If someone is really upset about this, they can always override /obj/hitby() here I guess.